Sources/SwiftDocC/Utility/ValidatedURL.swift

```diff
@@ -48,6 +55,16 @@ public struct ValidatedURL: Hashable, Equatable {
         self.components = components
     }
     
+    /// Creates a new RFC 3986 valid URL by using the given symbol destination.
+    ///
+    /// - Parameter symbolDestination: Source URL address as string
+    init(symbolDestination: String) {
```

Comment:
Nit: wondering if "symbol path" would be more appropriate? It wasn't immediately clear to me what "destination" referred to in this case